### PR TITLE
[Checkbox] Update example with hooks

### PR DESCRIPTION
**Pull request by KasparIsSo**
### WHY are these changes introduced?

See #1662

### WHAT is this pull request doing?

Updated the Checkbox example to use hooks instead of class syntax

### How to 🎩

Run yarn dev.
In StoryBook navigate to the checkbox example and confirm they still work.

### CODE REVIEW

File: src/components/Checkbox/README.md [hunk 1]
```diff
@@ -108,26 +108,17 @@ You agree to the Terms of Service
...
-        onChange={this.handleChange}
-      />
-    );
-  }
-
-  handleChange = (value) => {
-    this.setState({checked: value});
-  };
+function CheckboxExample() {
+  const [checked, setValue] = useState(false);
```
[the remark this answers is not in the document]
Comment (KasparIsSo) [reply src/components/Checkbox/README.md]: Makes a lot of sense to me!

**Comment by BPScott** [comment]
>  I'm not sure why it wants you to sign the CLA since your internal?

The commit author is what's checked and that's "author Kaspar So <kasparso@Kaspars-MacBook-Pro-2.local>", and that email can't be connected to a github account.

**Comment by KasparIsSo** [comment]
> > I'm not sure why it wants you to sign the CLA since your internal?
> 
> The commit author is what's checked and that's "author Kaspar So [kasparso@Kaspars-MacBook-Pro-2.local](mailto:kasparso@Kaspars-MacBook-Pro-2.local)", and that email can't be connected to a github account.

Yea I realized that my config was not setup correctly, I think we're all good to go now though!

**Comment by AndrewMusgrave** [comment]
> The commit author is what's checked and that's "author Kaspar So kasparso@Kaspars-MacBook-Pro-2.local", and that email can't be connected to a github account.

I was having a chat earlier with Kaspar and he was having some git issues which seem to be resolved now 👌 

@KasparIsSo would you mind adding an entry to `UNRELEASED.md`? Feel free to refresh my review when your ready!

**Comment by BPScott** [comment]
Seems we dropped the ball on making it clear that once a polaris team member give approval review then you should merge your own PRs.

We've since merged the version-4.0.0 branch into master so I will cherry pick this onto master.

**Comment by KasparIsSo** [comment]
> Seems we dropped the ball on making it clear that once a polaris team member give approval review then you should merge your own PRs.
> 
> We've since merged the version-4.0.0 branch into master so I will cherry pick this onto master.

Ahh, my mistake. Sorry about that and thanks.

**Comment by BPScott** [comment]
Cherry-picked as a596f0afd

**Pull request closed without merging**